econml/validate/results.py

```diff
@@ -22,11 +22,11 @@ class CalibrationEvaluationResults:
     def __init__(
         self,
         cal_r_squared: np.array,
-        df_plot: pd.DataFrame,
+        plot_dict: Dict[Any, pd.DataFrame],
```

Comment:
These aren't plots, so maybe something like `plot_data_dict` would be more appropriate; in any case, the docstring associated with the class hasn't been updated to match the new name here.